### ISSUE TITLE
Add `FindByAriaLabelAttribute`

### ISSUE THREAD

**Issue by YevgeniyShunevych** [atata-framework/atata#751]
`FindByAriaLabelAttribute` - specifies that a control should be found by the `aria-label` attribute. Finds the control that has the `aria-label` attribute matching the specified term(s). Uses `TermCase.Sentence` as the default term case.
